File: Aula03/Python/Retangulo.py
class Retangulo:

    def __init__(self, comprimento, largura):
        self.comprimento = comprimento
        self.largura = largura

    def calcular_area(self):
        area = self.comprimento * self.largura
        return area

    def calcular_perimetro(self):
        perimetro = 2 * (self.comprimento + self.largura)
        return perimetro

ret = Retangulo(200, 300)
area = ret.calcular_area()
perimetro = ret.calcular_perimetro()

print(f"area: {area}")
print(f"Perimetro: {perimetro}")

File: Aula03/Java/Retangulo.java
//criar classe retangulo 
//especificar sua area e seu perimetro

public class Retangulo {
    private double comprimento;
    private double largura;

    public Retangulo(double comprimento, double largura) {
        this.comprimento = comprimento;
        this.largura = largura;
    }

    public double calcularArea() {
        return comprimento * largura;
    }

    public double calcularPerimetro() {
        return 2 * (comprimento + largura);
    }

    public static void main(String[] args) {
        Retangulo ret = new Retangulo(200, 300);
        System.out.println("area: " + ret.calcularArea());
        System.out.println("perimetro: " + ret.calcularPerimetro());
    }
}
